test-aws/src/test/groovy/io/micronaut/starter/core/test/aws/CreateAwsLambdaS3NotificationEventSpec.groovy

```diff
@@ -0,0 +1,36 @@
+package io.micronaut.starter.core.test.aws
+
+import io.micronaut.starter.application.ApplicationType
+import io.micronaut.starter.options.BuildTool
+import io.micronaut.starter.options.Language
+import io.micronaut.starter.options.TestFramework
+import io.micronaut.starter.test.ApplicationTypeCombinations
+import io.micronaut.starter.test.CommandSpec
+import spock.lang.Unroll
+
+class CreateAwsLambdaS3NotificationEventSpec extends CommandSpec {
+
+    @Override
+    String getTempDirectoryPrefix() {
+        "test-aws-lambda-s3-event-notification"
+    }
+
+    @Unroll
+    void 'create-#applicationType with features aws-lambda-s3-event-notification #lang and #build and test framework: #testFramework'(ApplicationType applicationType,
+                                                                                                                             Language lang,
+                                                                                                                             BuildTool build,
+                                                                                                                             TestFramework testFramework) {
+        given:
+        List<String> features = ['aws-lambda-s3-event-notification']
+        generateProject(lang, build, features, applicationType, testFramework)
+
+        when:
+        String output = executeBuild(build, "test")
```

Comment:
The generated project here has no tests. Should it have?

Reply:
the handler returns `Void`. Thus, it is intentional the lack of generated test.